src/shared/components/unsupportedBrowserModal/UnsupportedBrowserModal.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import UnsupportedBrowserModal from "./UnsupportedBrowserModal";
+import browser from 'detect-browser';
+
+describe('UnsupportedBrowserModal', () => {
+
+    // let wrapper: any;
+    //
+    // before(()=> {
+    //
+    //      wrapper = shallow(<UnsupportedBrowserModal/>);
+    //
+    // });
+
+    it('does not show modal when Chrome is being used', () => {
```

Comment:
@Tamba32 The tests are actually run in PhantomJS, so this is currently not what is being tested. It is rather `shows modal when PhantomJS is being used`. But yeah would be good to add some tests where you specifically set the name of the browser, to verify that it works.